student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
The `initialize` method in `Student class` also takes age,  name = "unknown", and parent_permission = true as parameters. Kindly add these accordingly.